highfidelity/hifi
# 8495 No mirror camera in HMD mode

https://highfidelity.fogbugz.com/f/cases/8495
...

## TEST PLAN
- Make sure the menu option view/mirror doesn't appear while in HMD mode.
- Using the shortcut CTRL+H should change to mirror camera only in Desktop mode.  
- Change to mirror mode using this script:
`(function() {
	Camera.setModeString("mirror");
}());`
- You should be allow to change to mirror mode while in HMD.
